lighthouse-core/runner.js

```diff
@@ -108,8 +109,7 @@ class Runner {
         audits: resultsById,
         artifacts: runResults.artifacts,
         runtimeConfig: Runner.getRuntimeConfig(opts.flags),
-        score: report ? report.score : 0,
-        reportCategories: report ? report.categories : [],
+        reportCategories: categories,
```

Comment:
before this defaulted to an empty array. Will it cause problems anywhere as `undefined` (and removed from the JSON) instead?